RadarSDK/RadarAPIClient.m

```diff
@@ -138,6 +138,18 @@ - (void)flushReplays:(NSArray<NSDictionary *> *_Nonnull)replays
                            logPayload:NO
                       extendedTimeout:YES
                     completionHandler:^(RadarStatus status, NSDictionary *_Nullable res) {
+                            id eventsObj = res[@"events"];
+
+                            NSArray<RadarEvent *> *events = [RadarEvent eventsFromObject:eventsObj];
+                            if (events.count) {
+                                NSArray *eventsArr = (NSArray *)eventsObj;
+                                NSDictionary *lastEvent = [eventsArr lastObject];
+                                NSDictionary *userObj = lastEvent[@"user"];
```

Comment:
I think it's safer to keep it consistent with Android and read the user from the top level

Reply:
Oh yes, just didn't have this pushed up. https://github.com/radarlabs/radar-sdk-ios/pull/269/commits/8de0cb56f03163edcea54b3e8d82dcc02847179e